.github/workflows/android.yml

```diff
@@ -0,0 +1,27 @@
+name: cross compile with android-ndk
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  cross-compile-for-android:
+    strategy:
+      fail-fast: false
+      matrix:
+        target-abi: [armeabi-v7a, arm64-v8a, x86, x86_64]
+
+    runs-on: macos-10.15
```

Comment:
Why do we need to build on mac? It usually takes longer until one gets a mac machine for actions and they are slower than the linux counterparts

Reply:
I can change it to `Ubuntu-20.04`

Reply:
In the other workflow we use `ubuntu-latest`, so that should do it :)